CMakeLists.txt

```diff
@@ -323,25 +323,48 @@ set(CPACK_COMPONENT_INCLUDE_TOPLEVEL_DIRECTORY TRUE)
 # Compress package in parallel.
 set(CPACK_THREADS 0 CACHE STRING "")
 
-
-if(CMAKE_OSX_ARCHITECTURES)
-    # For universal binaries don't include architecture in the package name.
-    set(CPACK_SYSTEM_NAME "${CMAKE_SYSTEM_NAME}")
+# set processor_name
+string(TOLOWER ${CMAKE_SYSTEM_PROCESSOR} processor_name)
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    # If we're compiling on OSX we need to understand what targets we're
+    #  building for.
+    string(FIND "${CMAKE_OSX_ARCHITECTURES}" "x86_64" x86_result)
+    string(FIND "${CMAKE_OSX_ARCHITECTURES}" "arm64" arm64_result)
+
+    if((NOT ${x86_result} EQUAL -1) AND (NOT ${arm64_result} EQUAL -1))
+        set(processor_name "unified")
```

Comment:
Is it "universal"? https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary

Reply:
Ah yes, thanks! Done.